Add information page about crash reporting (#3497)
flutter/website: src/docs/reference/crash-reporting.md

---
title: Flutter crash reporting
---

If you have not opted-out of Flutter's analytics and crash reporting, when a
`flutter` command crashes it attempts to send a crash report to Google in order
to help Google contribute improvements to Flutter over time. A crash report
may contain the following information:

> * The name and version of your local operating system

Reviewer: ```suggestion
* The name and version of your local operating system.
```

> * The version of Flutter

Reviewer: ```suggestion
* The version of Flutter.
```

> * The runtime type of the error, e.g. `StateError` or `NoSuchMethodError`

Reviewer: ```suggestion
* The runtime type of the error, for example `StateError` or `NoSuchMethodError`.
```

Reviewer: https://developers.google.com/style/word-list#letter-i 
(see entry for avoiding using e.g.)

* The stack trace generated by the crash, which contains references to 
  the Flutter CLI's own code and contains no references to your application

>   code

Reviewer: ```suggestion
code.
```

* A stable client ID - a constant and unique number generated for the computer
  where Flutter is installed. It helps us deduplicate multiple identical crash
  reports coming from the same computer. It also helps us verify if a fix works
  as intended after you upgrade to the next version of Flutter.

Google handles all data reported by this tool in accordance with the 
[Google Privacy Policy][].

## Disabling analytics reporting

You can opt out of anonymous crash reporting and feature usage statistics from

> Flutter by running the following command.

Reviewer: ```suggestion
Flutter by running the following command:
```
